frontend/src/App.test.js

```diff
@@ -2,7 +2,7 @@ import { render, screen } from '@testing-library/react';
 import App from './App';
 
 test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+    render(<App />);
+    const linkElement = screen.getByText(/learn react/i);
```

Comment:
This test will fail ;)

Reply:
i do hope it makes sense to do FR testing at the very end.... ^^'

Reply:
👍 a failing test will break your ci pipeline, so i suggest you delete this file